site/en/docs/extensions/mv3/architecture-overview/index.md

```diff
@@ -36,11 +36,10 @@ as the most important files and the capabilities the extension might use.
     "128": "icon_128.png"
   },
   "background": {
-    "persistent": false,
-    "scripts": ["background_script.js"]
+    "service_worker": "background.js"
   },
-  "permissions": ["https://*.google.com/", "activeTab"],
```

Comment:
Shouldn't the url regex permissions `https://*.google.com/` be moved to a `host_permissions`  key?